api/workAllocation2/util.ts

```diff
@@ -537,14 +541,26 @@ export function mapRoleCaseData(roleAssignment: RoleAssignment, caseDetail: Case
   };
 }
 
-export function getCaseTypesFromRoleAssignments(roleAssignments: RoleAssignment[]): string {
-  const caseTypes = roleAssignments
-    .filter((roleAssignment: RoleAssignment) => roleAssignment.attributes && roleAssignment.attributes.caseType)
-    .map((roleAssignment: RoleAssignment) => roleAssignment.attributes.caseType)
-    .reduce((query: string, caseType: string) => {
-      return query.includes(caseType) ? query : `${query}${caseType},`;
-    }, '');
-  return caseTypes[caseTypes.length - 1] === ',' ? caseTypes.slice(0, caseTypes.length - 1) : caseTypes;
+export function getCaseTypesFromRoleAssignments(roleAssignments: RoleAssignment[]): any {
```

Comment:
please return a type instead of any

Reply:
like I said this was all a WIP, i've yet to tidy up. Thought you only wanted the PR just to see how the data was being handled